src/kinetics/Kinetics.cpp

```diff
@@ -703,13 +703,24 @@ bool Kinetics::addReaction(shared_ptr<Reaction> r, bool resize)
         m_ready = false;
     }
 
+    for (const auto& [id, callback] : m_reactionChangedCallbacks) {
+        callback();
+    }
```

Comment:
Based on where this is called, and how it is used in conjunction with `PlasmaPhase`, it might be more accurate to name this a `reactionAddedCallback`.